### PR TITLE
Ch #160441415 docstrings

### PR DESCRIPTION
**Pull request by milamish**
what does this PR do?
this PR adds docstrings.

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from users.views import users
 from queries.views import queries
 from views import main
+from models import *
```
Comment (ndero) [inline app/__init__.py]: please avoid importing `*` . Instead prefer specific imports.

File: app/models.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 cursor = connection.cursor()
 
 def table():
-	connection= psycopg2.connect(host = 'localhost', user = 'postgres', password = 'milamish8', dbname = 'stack')
+	connection = psycopg2.connect(host = 'localhost', user = 'postgres', password = 'milamish8', dbname = 'stack')
```
Comment (ndero) [inline app/models.py]: there should be no space between function parameters and their default values. i.e `...connect(host='localhost', user='postgres', ...)`

File: app/run.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
 from __init__ import *
```
Comment (ndero) [inline app/run.py]: You should never import from `__init__` import instead from the module that has been initialized with the `__init__.py` file

File: app/users/views.py [hunk 4]
```diff
@@ -1,102 +1,98 @@
...
 from flask_restful import Resource
-from functools import wraps
-import psycopg2
-import re
-import hashlib
-import jwt
-import datetime
 
 from __init__ import *
 from models import *
```
Comment (ndero) [inline app/users/views.py]: Again be specific, avoid these wildcard imports. Only import what you need.

File: app/__init__.py (hunk 1 above)
Comment (milamish) [reply app/__init__.py]: changing that

File: app/run.py (hunk 3 above)
Comment (milamish) [reply app/run.py]: doing that

**Pull request merged**